src/main/java/com/example/parser/utils/Parser.java

```diff
@@ -1,38 +1,47 @@
 package com.example.parser.utils;
 
+import com.example.parser.dto.ResponseFromParsing;
 import com.example.parser.entity.PermitForEmissionsOfPollutants;
 import com.example.parser.repository.PermitRepository;
+import com.example.parser.service.PermitService;
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.FileReader;
-import java.io.IOException;
+import java.io.*;
 import java.util.*;
 import java.util.concurrent.TimeUnit;
 
+
 public class Parser {
```

Comment:
This class must parse needed data. The code shouldn't think about downloading the data or preparation it for saving into database.

Reply:
done